### ISSUE TITLE
Adding sub sport types to both medal routes

### ISSUE THREAD

**Issue by GToidZ** [SPaM-Skill-Issue/sota-backend#11]
Currently, our medal routes `/medal/c/:country_code` and `/medal/s/:sport_id` does not contain information on medal for each sub sport type, making it impossible to really know which country gets medal on on which event. The changes we need to make:

* Add a new chunk of information under the `individual_sports` or `individual_countries`. That new information is a Javascript object with the following format as proposed in the Discord but modified a little bit:
* For `/medal/s/:sport_id`,
  ```javascript
  individual_countries: [
      {
         country_code: "FR",
         country_name: "France",
         gold: 10,
         silver: 3,
         bronze: 7,
         sub_sports: [
             {
                sub_id: 1,
                sub_name: "100m",
                gold: 1,
                silver: 1,
                bronze: 0,
             }, ...
         ]
      },
  ...
  ]       
  ```
* For `/medal/c/:country_code`,
  ```javascript
  individual_sports: [
      {
         sport_id: 3,
         sport_name: "Atheletics",
         gold: 10,
         silver: 3,
         bronze: 7,
         sub_sports: [
             {
                sub_id: 1,
                sub_name: "100m",
                gold: 1,
                silver: 1,
                bronze: 0,
             }, ...
         ]
      },
  ...
  ]       
  ```
* The reason to put the chunk into under the `individual_*` object is that we can reduce fatigue in redesigning the response schema.

**Comment by Supakrit65** [comment]
`D` for `Done` | `C` for `Complete` | `M` for `Monkey`
![](https://media.tenor.com/I56ob4FH56wAAAAd/monkey-fat-monkey.gif)

**Comment by GToidZ** [comment]
Closed by #10 and #12
